src/qt/recentrequeststablemodel.cpp

```diff
@@ -139,10 +139,9 @@ bool RecentRequestsTableModel::removeRows(int row, int count, const QModelIndex
 
     if(count > 0 && row >= 0 && (row+count) <= list.size())
     {
-        const RecentRequestEntry *rec;
         for (int i = 0; i < count; ++i)
         {
-            rec = &list[row+i];
+            const RecentRequestEntry *rec = &list[row+i];
```

Comment:
`RecentRequestEntry* rec`

Reply:
cc @Empact :-)

Reply:
Ah I didn't mean to remove const, sorry, just the space..